### DISCUSSION TITLE
Anchor links for headers

### DISCUSSION THREAD

**Discussion by erlend-sh** [writefreely/writefreely#574] [Enhancements]
I'd like to have anchor links for headers, like in GitHub readmes. ['Open Source Explained'](https://blog.erlend.sh/open-source-explained) is composed of a series of posts combined into one because they're all interconnected. But sometimes I'd still like to link to a specific piece, i.e. essentially an individual post within the larger work.

**Comment by tezoatlipoca** [comment]
You can accomplish this already, although not natively in Markdown. To use anchor links you have to get an id attribute in the rendered header HTML tag. So either:

Use markdown to define the header, then embed the id in a span tag, like:

`# <span id="section12">Section 12</span>`

OR skip the markdown and define the header using straight HTML

`<h1 id="section12">Section 12</h1>`

**Reply by erlend-sh** [reply]
Oh, you're right. I've just updated the article with a ToC, thanks a bunch! ❤️
